code/modules/research/designs.dm

```diff
@@ -230,6 +230,15 @@ other types of metals and chemistry for reagents).
 	build_path = /obj/item/weapon/storage/part_replacer
 	sort_string = "CBAAA"
 
+/datum/design/item/stock_part/BRPED
```

Comment:
i should have picked up on this the first time, but if possible, this should be moved to code/modules/urist/datums/uristdesigns.dm to save further merging headaches. Besides that, code looks good.